jax/experimental/jax2tf/README.md

```diff
@@ -1,30 +1,35 @@
 # JAX and TensorFlow interoperation (jax2tf/call_tf)
 
 This package provides experimental support for interoperation between JAX and TensorFlow.
+There are two interoperation directions: 
+(a) `jax2tf.convert`: using JAX functions in a TensorFlow context, e.g., 
+for eager or graph execution, or for saving as a SavedModel, 
+and (b) `call_tf`: using TensorFlow 
+functions in a JAX context, e.g., to use a TensorFlow library or a SavedModel. 
```

Comment:
Just a suggestion @gnecula: list the 2 items to make them stand out a little (I think the users will be scanning the text mostly and these 2 interop directions are good to pay attention to)

```suggestion
There are two interoperation directions: 

- `jax2tf.convert`: using JAX functions in a TensorFlow context, e.g., 
for eager or graph execution, or for saving as a SavedModel; and 
- `call_tf`: using TensorFlow  functions in a JAX context, e.g., to use a
TensorFlow library or a SavedModel. 
```

or something like that